UpdateDateTime uses a native method

MahmoudGSaleh opened the pull request:
UpdateDateTime uses DateTime.Now, which uses a cached time zone. So,
when the time zone is changed, this is not reflected in the current date
time and a restart is required. To prevent that, we use a native method,
GetLocalTime() instead.

MahmoudGSaleh committed:
UpdateDateTime uses a native method

UpdateDateTime uses DateTime.Now, which uses a cached time zone. So,
when the time zone is changed, this is not reflected in the current date
time and a restart is required. To prevent that, we use a native method,
GetLocalTime() instead.

msftclas commented:
Hi **@msalehmsft**, I'm your friendly neighborhood Microsoft Pull Request Bot (You can call me MSBOT). Thanks for your contribution!
    <span>You've already signed the contribution license agreement. Thanks!</span>
        <p>The agreement was validated by Microsoft and real humans are currently evaluating your PR.</p>

TTYL, MSBOT;

MahmoudGSaleh committed:
Make OOBE screen content visible on small screens

Enclosing the OOBE screen content within a ScrollViewer enables access
to the UI elements including language settings and "Next" button

MahmoudGSaleh committed:
Ensure OOBE screen's UI is responsive
